yarn-project/end-to-end/src/e2e_deploy_contract/private_initialization.test.ts

```diff
@@ -29,6 +31,16 @@ describe('e2e_deploy_contract private initialization', () => {
     expect(txEffects!.data.nullifiers).toContainEqual(expected);
   });
 
+  // Tests calling a private function in an uninitialized and undeployed contract.
+  // Requires registering the contract artifact and instance locally in the pxe.
+  // This contract does not have a constructor, so the fn does not need the noinitcheck flag.
+  it('executes a function in a contract without initializer', async () => {
```

Comment:
Would update [this comment](https://github.com/AztecProtocol/aztec-packages/blob/a75ff87e1b143e2a527c55006923812b919ca180/noir-projects/noir-contracts/contracts/docs_example_contract/src/main.nr#L96) with information that this test also relies on the contract not having an initializer.

Reply:
The bigger issue is to abuse `DocsExample` in this way - we should have contracts with specific properties that we want to test.

Reply:
Since I am already working on tests today I could do that change.